pkgs/tools/networking/network-manager/applet.nix

```diff
@@ -20,8 +20,18 @@ in stdenv.mkDerivation rec {
       url = https://gitlab.gnome.org/GNOME/network-manager-applet/merge_requests/12.patch;
       sha256 = "0q5qbjpbrfvhqsprnwjwz4c42nly59cgnbn41w2zlxvqf29gjvwk";
     })
+    (fetchpatch {
+      # https://gitlab.gnome.org/GNOME/network-manager-applet/issues/11
+      # should be fixed in 1.8.15
+      name = "show-import-file-chooser.patch";
+      url = https://gitlab.gnome.org/GNOME/network-manager-applet/merge_requests/15.patch;
+      sha256 = "11913cin514jgg9qg9iz8isyj5jhj6393b7a56hqc6gprbkyszwq";
+    })
   ];
 
+  # remove again if #11 is merged
```

Comment:
Do you mean [!11](https://gitlab.gnome.org/GNOME/network-manager-applet/merge_requests/11)?

Reply:
explicitly linked https://gitlab.gnome.org/GNOME/network-manager-applet/merge_requests/15 now.